### PR TITLE
class01 lab refactoring

### PR DESCRIPTION
**Pull request by kikobr82**
# Description
Refactor of class01 aws labs.

Student Name:

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [X] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-aws01/ANSWER.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Commands to connect to the second EC2 instance:
+```
+Add your commands and their outputs here
+```
+
+<!-- Don't change anything below this point-->
+<!-- Before commiting, remove both commented lines--> 
+***
+Answer for exercise [c01-aws01](TBD)
```
Comment (denstorti) [inline classes/01class/exercises/c01-aws01/ANSWER.md:28]: to be updated with commit sha. Exercise content is OK.

File: classes/01class/exercises/c01-aws02/README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 
+## Submit a PR with the folowing file:
+- README.md based on the [ANSWER.md](ANSWER.md) file with the commands requested. 
+
+## References
+- [S3 Bucket Creation](https://docs.aws.amazon.com/cli/latest/reference/s3/mb.html)
+
 
 ### Tip:
 You can access the file on the S3 bucket by creating an IAM role with S3 policy and attaching it to your EC2 instance. One of the best practices when creating IAM policies is to grant least privilege to the users. The objective is to initially give the users only the access/permission they need to complete the intended task and grant additional permissions as necessary later. This is more secure than starting with permissions that are too lenient and then trying to tighten them later.
```
Comment (denstorti) [inline classes/01class/exercises/c01-aws02/README.md:15]: Maybe it is better to tell them which managed policy to use as the IAM class is just later in the course

File: classes/01class/exercises/c01-aws02/ANSWER.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add a brief descrition of the challenges you faced:
+```
+Add comments here
+```
+
+<!-- Don't change anything below this point-->
+<!-- Before commiting, remove both commented lines--> 
+***
+Answer for exercise [c01-aws02](TBD)
```
Comment (denstorti) [inline classes/01class/exercises/c01-aws02/ANSWER.md:28]: Commit to be updated.

File: classes/01class/exercises/c01-aws04-deprecated/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Tools in use (C01-E01)
+# Tools in use (c01-aws04 - deprecated)
```
Comment (denstorti) [inline classes/01class/exercises/c01-aws04-deprecated/README.md:1]: Should remove this from tracked lab exercises in `/scripts/labs.txt`?

Comment (kikobr82) [reply classes/01class/exercises/c01-aws04-deprecated/README.md:1]: I'm removing that in the next PR.

File: classes/01class/exercises/c01-aws02/README.md (hunk 2 above)
Comment (kikobr82) [reply classes/01class/exercises/c01-aws02/README.md:15]: We can do it, but I reckon this part is added by one of the students in the past.
I'll make this adjustment anyway to make things clearer..

Comment (kikobr82) [reply classes/01class/exercises/c01-aws02/README.md:15]: done

**Comment by denstorti** [comment]
Closes #19

**Pull request merged**